docs/subnets/hello-world-precompile-tutorial.md

```diff
@@ -183,10 +200,12 @@ npm install -g solc
 npm install -g yarn
 ```
 
-### Official GitHub Tutorial
+### Complete Code
 
-Please also use the [official GitHub tutorial](https://github.com/ava-labs/hello-world-official-precompile-tutorial/pull/1)
-to double check your work as well.
+You can inspect the [Hello World Precompile tutorial](https://github.com/ava-labs/hello-world-official-precompile-tutorial/pull/1)
+for the complete code.
+
```

Comment:
did we forget to change the link to helloworld-official-tutorial-v2 on line 205?

Reply:
updated with diff